### DISCUSSION TITLE
Information about the upcoming version 1.17

### DISCUSSION THREAD

**Discussion by alexcimander** [Digitaler-Impfnachweis/covpass-ios#108] [Announcements]
📢 **Information about the upcoming version 1.17** 📢  
Version 1.17 of the CovPass and CovPassCheck is planned to be released early february. It will include a variety of features and fixes that were requested by the community.

**Planned Main Features CovPass:**
•	Improvements on Trustlist-Updates: The apps will now use etag, gzip and other http-features to improve the update-mechanisms of the trustlist.

**Planned Main Features CovPassCheck:**
•	The CovPassCheck now offers scan-sequences to check your 2G+ Status. The awaited booster-recognition, especially regarding the use-case “valid test not necessary after booster vaccination” is planned to be added with the next release 1.18.

**Planned Bug-Fixes:**
•	Multiple UI-Bugfixes as Accessibility-Enhancements are planned for the release.  
  
Thank you all for your feedback, ideas and critics!

stay safe and best regards,

Team CovPass and CovPassCheck ✔️

**Comment by timokoenig** [comment]
@alexcimander are these information for iOS and Android? The reason I am asking is because Android already implemented the etag improvement in the latest release https://github.com/Digitaler-Impfnachweis/covpass-android/issues/89#issuecomment-1016562573
